src/igvfd/schemas/changelogs/curated_set.md

```diff
@@ -0,0 +1,3 @@
+## Changelog for *`curated_set.json`*
+
+### Minor changes since schema version 1
```

Comment:
I don't think there are changes so the line about minor changes shold not be there?